ouroboros-network/src/Ouroboros/Network/AnchoredFragment.hs

```diff
@@ -785,13 +822,17 @@ mapAnchoredFragment f (AnchoredFragment a c) =
     AnchoredFragment (castAnchor a) (CF.mapChainFragment f c)
 
 -- | \( O\(n\) \). Variation on 'filterWithStop' without a stop condition.
-filter :: forall block. HasHeader block
-       => (block -> Bool)  -- ^ Filtering predicate
-       -> AnchoredFragment block
-       -> [AnchoredFragment block]
-filter p = filterWithStop p (const False)
+filter ::
+     forall block. HasHeader block
+  => (block -> Bool)  -- ^ Filtering predicate
+  -> AnchoredFragment block
+  -> [AnchoredFragment block]
+filter p = filterWithStopSpec p (const False)
```

Comment:
Oh, oops, this should be `filterWithStop`, this is a left over from testing. Will fix.